Branch-SDK/src/io/branch/referral/SystemObserver.java

```diff
@@ -448,9 +448,6 @@ public int getUpdateState(boolean updatePrefs) {
         String currAppVersion = getAppVersion();
         if (PrefHelper.NO_STRING_VALUE.equals(pHelper.getAppVersion())) {
             // if no app version is in storage, this must be the first time Branch is here
```

Comment:
![screen shot 2016-12-07 at 1 24 47 pm](https://cloud.githubusercontent.com/assets/3870241/20987248/be595b26-bc80-11e6-8cae-7d88eb7ee384.png)
Yeah it's literally never used, removing.